### ISSUE TITLE
Question : Where is libzkpp_fake CMake target / 

### ISSUE THREAD

**Issue by GuillaumeDua** [tgockel/zookeeper-cpp#121]
Where is the CMake target mentioned in the [documentation](https://github.com/tgockel/zookeeper-cpp#zkfake) ?
Not implemented yet ?

I'd like to use it to implement some UTs.

Ps : Cannot set a label ?

**Comment by tgockel** [comment]
Yes, it has not yet been implemented. I'm holding off on implementation until the real version of the API is a bit more solidified.
